libs/application/templates/driving-license/src/forms/draft/subSectionSummary.ts

```diff
@@ -159,23 +159,39 @@ export const subSectionSummary = buildSubSection({
         buildDividerField({}),
         buildKeyValueField({
           label: m.overviewPaymentCharge,
-          value: ({ externalData, answers }) => {
-            const items = externalData.payment.data as {
+          value: (application) => {
+            const items = application.externalData.payment.data as {
               priceAmount: number
               chargeItemCode: string
             }[]
+
             const targetCode =
-              answers.applicationFor === B_TEMP
+              application.answers.applicationFor === B_TEMP
                 ? 'AY114'
-                : answers.applicationFor === BE
+                : application.answers.applicationFor === BE
                 ? 'AY115'
+                : application.answers.applicationFor === B_FULL_RENEWAL_65
+                ? 'AY113'
                 : 'AY110'
```

Comment:
Do we want to create a function for this decider code so that we can maintain it more clearly elsewhere? Like in form/utils or something?

Also, can this not be simplified with something like:
```ts
const codeLookup: Record<string, string> = {
   [B_TEMP]: 'AY114',
   [BE]: 'AY115',
   [B_FULL_RENEWAL_65]: 'AY113'
}
 ```
 
 So that it can be accessed with something like
 ```ts
 const targetCode = codeLookup[application.answers.applicationFor] ?? 'AY110'
 ```
 
 Just wondering if that makes sense and if that creates a more maintainable base :)